### ISSUE TITLE
Stripe Connect, Charge connected account via platform

### ISSUE THREAD

**Issue by Tarek-AbdelMaqsoud** [jaymedavis/stripe.net#547]
Hello,
 Actually i need to use stripe in an E-Commerce website.
 the scenario i need is as the following
 
 1- Service receiver login to his account and pay to the service 
 2- the payment is totally transferred to the website account
 3- the service provider is confirming the payment done
 4- the amount is totally sent to the service provider deducting the website's fees
 
 I am stuck into the step 4, as i followed the following article https://stripe.com/docs/recipes/store-builder
 
 and it helped a lot, but when transferring money from my platform to the connected account, the amount transferred is added twice, in my platform's account and the user's account 
 
 Any help

**Comment by jaymedavis** [comment]
Hmm... I'm not sure, can you create a Gist of the code? It sounds like you may need to contact Stripe support and verify your logic as well, but we can make sure it's nothing with the code first.

**Comment by Tarek-AbdelMaqsoud** [comment]
Well, my code is 


```
var myCharge = new StripeChargeCreateOptions();
myCharge.Amount = myAmount;

myCharge.Currency = "USD";
myCharge.Source = new StripeSourceOptions();
myCharge.Source.TokenId = myTokenId; 
myCharge.Destination = connectedAccountId;


var chargeService = new StripeChargeService();
chargeService.ApiKey = myAPiKey;
StripeCharge stripeCharge = chargeService.Create(myCharge);
```

I am not sure of the token that i passed, as i have passed the token that i have used to charge the service receiver at the beginning which of course seems not logic

If so, what token shall i pass ?!

After reviewing the stripe accounts for both service provider and my platform, if found that the service provider's account has been loaded with the amount i passed (Source is my platform)

And the platform account has been added the same amount i sent, instead of withdrawing it.

**Comment by jaymedavis** [comment]
Stripe is phasing out recipients, so I would ditch that destination. Try to setup the connected account on the StripeRequestOptions parameter to the service instead.

**Comment by jaymedavis** [comment]
Basically, remove the line for destination on the charge, and pass the connectedAccountId in a StripeRequestOptions object as the second parameter to the charge service call- make sense?

**Comment by Tarek-AbdelMaqsoud** [comment]
Great, it worked fine when replacing the destination with the StripeRequestOptions  

Thank you so much,

**Comment by jaymedavis** [comment]
No worries. Glad you are all set.

**Comment by vbwyrde** [comment]
Does there happen to be a working example of this in the GitHub code base by chance?  I've tried following the directions in this thread but the code presented above doesn't actually work (at least in vb.net, which shouldn't make a difference once translated) so I'm thinking if I can just see one example of it working I should be able to get this going.  Thanks.

**Comment by remi-stripe** [comment]
@vbwyrde Not sure which part isn't working but our support team would be happy to help you! Since you are not reporting a bug on the library itself and this issue is over 18 months old, it does not seem like the best place to get help.

Please contact our support team directly here: https://support.stripe.com/email

**Comment by vbwyrde** [comment]
Ok Remi-stripe, thanks.  What I really was looking for is a working example of the relevant code to creating a charge in stripe connect for a customer who purchases a product from our platform vendor and sends the application fee to the platform owner.  I eventually figured it out through trial and error and hunting down little bits of code snippets online until I finally got a working version.  My point being - it would be tremendously helpful to have a stripe.net set of examples that show how to do this, and without it developers like me are just hunting and pecking our way to success... not optimal, but yes, it's not a bug that I am reporting.  Anwyay, thanks for the reply, and thanks for the stripe.net framework, which is great.  Much appreciated!
